### PR TITLE
Live container migration with docker-runc and criu

### PR DESCRIPTION
**Pull request by talsch**
Content for docker-runc, including live container migration with runc and CRIU, and 2 added ops for the base cp too

### CODE REVIEW

File: content/io/cloudslang/base/math/round.sl [hunk 1]
```diff
@@ -0,0 +1,40 @@
+#   (c) Copyright 2014 Hewlett-Packard Development Company, L.P.
```
Comment (mihaitusa) [inline content/io/cloudslang/base/math/round.sl]: Replace it with: (c) Copyright 2016 Hewlett Packard ...

File: content/io/cloudslang/base/math/round.sl [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+#   All rights reserved. This program and the accompanying materials
+#   are made available under the terms of the Apache License v2.0 which accompany this distribution.
+#
+#   The Apache License is available at
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+####################################################
+#!!
+#! @description: Checks if one percentage is less than another.
```
Comment (mihaitusa) [inline content/io/cloudslang/base/math/round.sl]: @talsch The description doesn't seems to be clear enough... I think isn't just checking... but round it right?

File: content/io/cloudslang/base/os/linux/check_linux_cpu.sl [hunk 3]
```diff
@@ -0,0 +1,85 @@
+#   (c) Copyright 2014 Hewlett-Packard Development Company, L.P.
```
Comment (mihaitusa) [inline content/io/cloudslang/base/os/linux/check_linux_cpu.sl]: @talsch Maybe rename it to check_linux_cpu_usage.sl WDYT?

File: content/io/cloudslang/docker/runc/checkpoint.sl [hunk 4]
```diff
@@ -0,0 +1,97 @@
+#   (c) Copyright 2014 Hewlett-Packard Development Company, L.P.
```
Comment (mihaitusa) [inline content/io/cloudslang/docker/runc/checkpoint.sl]: @talsch Maybe rename it to create_container_checkpoint WDYT?

File: cloud-slang-content [hunk 5]
```diff
@@ -0,0 +1 @@
+Subproject commit 85799de62821cc7ca02afdff9c59cf5d03e37018
```
Comment (mihaitusa) [inline cloud-slang-content]: @talsch  I think that should be removed.

File: content/io/cloudslang/docker/runc/checkpoint.sl [hunk 6]
```diff
@@ -0,0 +1,97 @@
...
+
+imports:
+  ssh: io.cloudslang.base.ssh
+  files: io.cloudslang.base.files
+  comparisons: io.cloudslang.base.comparisons
+flow:
+  name: checkpoint
+  inputs:
+    - docker_host
+    - port: "22"
```
Comment (mihaitusa) [inline content/io/cloudslang/docker/runc/checkpoint.sl]: @talsch  To be consistent with the rest of the content here should be: 
- port:
    default: "22"
    required: false

File: content/io/cloudslang/base/os/linux/check_linux_cpu.sl (hunk 3 above)
Comment (talsch) [inline content/io/cloudslang/base/os/linux/check_linux_cpu.sl]: will change to cpu utilization. cpu usage is different...

File: content/io/cloudslang/base/math/round.sl (hunk 1 above)
Comment (talsch) [inline content/io/cloudslang/base/math/round.sl]: will be changed for all flows/ops

File: content/io/cloudslang/docker/runc/examples/migrate.sl [hunk 7]
```diff
@@ -0,0 +1,236 @@
+#   (c) Copyright 2014 Hewlett-Packard Development Company, L.P.
+#   All rights reserved. This program and the accompanying materials
+#   are made available under the terms of the Apache License v2.0 which accompany this distribution.
+#
+#   The Apache License is available at
+#   http://www.apache.org/licenses/LICENSE-2.0
+####################################################
+
+
+######################################################## How to Prapare your infrastructure for this example ####################################
```
Comment (mihaitusa) [inline content/io/cloudslang/docker/runc/examples/migrate.sl]: @talsch  Add this section into descriptions area with @prerequisites tag in the same manner as it is in /vmware/virtual_machines/clone_virtual_machine for e.g.

**Comment by mihaitusa** [comment]
@talsch  Please handle the review comments!

File: content/io/cloudslang/base/math/round.sl (hunk 2 above)
Comment (talsch) [inline content/io/cloudslang/base/math/round.sl]: yes, wil be fixed

File: content/io/cloudslang/docker/runc/checkpoint.sl (hunk 4 above)
Comment (talsch) [inline content/io/cloudslang/docker/runc/checkpoint.sl]: the term is checkpoint a container. so I'll change it to checkpoint_container.
Will add the _container to the restore too...

**Pull request merged**